openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -276,3 +276,39 @@ def get_queryset(self):
             )
 
         return invitations
+
+
+class UpdatePermissions(ListView):
+    """
+    UpdatePermissions View
+    """
+    def get(self, request, *args, **kwargs):
+        """
+        It will get invitation object of the user and give permissions which
+        are assigned to that user in invitation object
+
+        Returns:
+            It will redirect to admin dashboard if user is active and have permissions
+            otherwise it will redirect to home page
+        """
+        user = self.request.user
+        invitation_obj = Invitation.objects.filter(email=user.email).first()
+
+        if not invitation_obj:
+            return redirect('/')
+
+        if invitation_obj.permission == Invitation.BU_ADMIN:
+            group = invitation_obj.business_unit.group
+        else:
+            group = Group.objects.get(name=ADG_ADMIN_GROUP_NAME)
+
+        user.groups.add(group)
+        user.is_staff = True
+        user.save()
+
+        invitation_obj.status = Invitation.ACCEPTED
+        invitation_obj.save()
```

Comment:
Let's write a method in Invitation model to change the status to Accepted and call that method here by passing the invitation_obj